modules/core/src/main/java/org/apache/synapse/core/axis2/Axis2SynapseEnvironment.java

```diff
@@ -235,21 +237,29 @@ private void setSeverUnitTestMode(ServerContextInformation contextInformation) {
             //get unit test received port
             String requestPort = System.getenv("UNIT_TEST_PORT");
```

Comment:
Should this be an environment variable? Or a system property? What happens if we decide to run 2 instances in the same machine?

Reply:
IMO 1e need to have it as a system property for the docker images to work properly.

Reply:
Changed these to system properties.